cli/tests/pcluster/api/controllers/test_cluster_operations_controller.py

```diff
@@ -938,7 +943,8 @@ def test_incompatible_version(self, client, mocker):
                             "StackName": "fake-name",
                             "ResourceType": "AWS::CloudFormation::WaitCondition",
                             "ResourceStatus": "CREATE_FAILED",
-                            "ResourceStatusReason": "WaitCondition received failed message: Fail to run preinstall",
+                            "ResourceStatusReason": "WaitCondition received failed message: Failed to execute "
+                            "OnNodeStart script...",
```

Comment:
nitpick, why there are 3 `.` here?

Reply:
The CFN failure reason string contains `OnNodeStart script`, it can also contains something else. I add `...` here to validate that the code should work with something else inside the string